### DISCUSSION TITLE
Big Plant Organism

### DISCUSSION THREAD

**Discussion by 40Eridani** [MaxRobinsonTheGreat/EvolutionSimulatorV2#138] [General]
I made this plant organism that has 873 cells, it's my biggest so far. If anyone has a larger cell, I'd love to see it! (or if you have any interesting ones)
![873cell](https://user-images.githubusercontent.com/108262639/175903846-ed8fb180-0dce-4523-801a-6b874602b5d9.png)

**Comment by AxoWasTaken** [comment]
What the heck?!!? How did you make an organism so large? I turn up my mutation rate ridiculously high and they grow to at maximum 100 cells until the population drops drastically and they genetically deteriorate.
